st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -104,80 +97,85 @@ def match(self, action_alias_match_api, **kwargs):
                                                     matches=[],
                                                     command=command)
             return [self._match_tuple_to_dict(match) for match in matches]
-        except (ActionAliasAmbiguityException) as e:
+        except ActionAliasAmbiguityException as e:
             LOG.exception('Command "%s" matched (%s) patterns.', e.command, len(e.matches))
-            pecan.abort(http_client.BAD_REQUEST, str(e))
-            return [self._match_tuple_to_dict(match) for match in e.matches]
+            return abort(http_client.BAD_REQUEST, str(e))
 
-    @request_user_has_permission(permission_type=PermissionType.ACTION_ALIAS_HELP)
-    @jsexpose(arg_types=[str, str, int], body_cls=ActionAliasHelpAPI,
-    status_code=http_client.ACCEPTED)
-    def help(self, action_alias_help_api, **kwargs):
+    def help(self, filter, pack, limit, offset, **kwargs):
```

Comment:
Yeah, I like those arguments being explicitly declared more more than previous approach.

Having said that, those `limit` and `offset` arguments / query params should really be global since they apply to every "list" / get_all controller, right?